### PR TITLE
[3.15.x] cf-secret

### PR DESCRIPTION
**Pull request by vpodzime**
Merge together:
https://github.com/cfengine/core/pull/4204
https://github.com/cfengine/enterprise/pull/620
https://github.com/cfengine/buildscripts/pull/750

### CODE REVIEW

**Comment by basvandervlies** [comment]
+1

**Comment by vpodzime** [comment]
@cf-bottom jenkins with exotics, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5210/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5210/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5210/

**Comment by vpodzime** [comment]
@cf-bottom jenkins with exotics, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5211/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5211/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5211/

File: tests/acceptance/testall [hunk 1]
```diff
@@ -100,6 +100,7 @@ CF_PROMISES=${CF_PROMISES:-}        ; export CF_PROMISES
 CF_SERVERD=${CF_SERVERD:-}          ; export CF_SERVERD
 CF_EXECD=${CF_EXECD:-}              ; export CF_EXECD
 CF_KEY=${CF_KEY:-}                  ; export CF_KEY
+CF_KEYCRYPT=${CF_KEYCRYPT:-}        ; export CF_KEYCRYPT
```
Comment (olehermanse) [inline tests/acceptance/testall:103]: Here and multiple places below, CF_KEYCRYPT is still mentioned. Can we change all of them to cf-secret? Might be the same in master?

Comment (vpodzime) [reply tests/acceptance/testall:103]: Yeah. I noticed that too.

Comment (vpodzime) [reply tests/acceptance/testall]: Done. Added a commit here and the *master* PR is here: https://github.com/cfengine/core/pull/4205

**Comment by olehermanse** [comment]
@cf-bottom jenkins with exotics, please.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5220/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5220/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5220/

**Pull request merged**